modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -680,9 +680,11 @@ else if (value instanceof Object[]) {
 
 			if (indexType.equals("keyword")) {
 				document.addKeywordSortable(name, valuesString);
+				document.addKeyword(_getSortableFieldName(name), valuesString);
```

Comment:
@p-albuquerque do we still need the `document.addKeywordSortable(name, valuesString);`?